src/StatusQ/Core/Theme/StatusDarkTheme.qml

```diff
@@ -68,6 +68,30 @@ ThemePalette {
         source: "../../../assets/fonts/InterStatus/InterStatus-Black.otf"
     }
 
+    property QtObject codeFont: FontLoader {
+        source: "../../../assets/fonts/RobotoMono/RobotoMono-Regular.ttf"
```

Comment:
I needed just Regular. Adding others would increase deployment size. I hope that's Okay.

Reply:
I'll just keep those and if we feel this is a bottle neck it terms of size we can still remove unused font sources.